swagger.yaml

```diff
@@ -1,83 +1,401 @@
 swagger: '2.0'
 info:
-  title: Pazuzu registry
-  description: Generating Dockerfiles by sets of required functions
-  version: "0.0.1"
+  title: Pazuzu api specification
+  description: Pazuzu api specification - what need to be done to create pazuzu cli and ui.
+  version: "0.0.2"
+host: pazuzu.zalando.net
 schemes:
   - http
-basePath: /
 produces:
   - application/json
 paths:
-  /features:
+  /api/features:
     get:
-      summary: Array of all available Features
+      summary: List all the features in pazuzu
       description: |
-        The features endpoint returns information about all available features in the registry. The response is an array of feature_refs containing the features unique name and a short description.
+        Returns all the features without it's dependencies. Each features can further be accessed
+        one by one and the dependencies will be there
+      parameters:
+        - name: name
+          in: query
+          description: Optional value, that must present in feature name.
```

Comment:
Optional here is redundant by the `required: false` I guess.
Also the description should be more like "search for this string on feature name".